helm-charts/common/llm-uservice/templates/configmap.yaml

```diff
@@ -19,7 +19,14 @@ data:
   vLLM_ENDPOINT: "http://{{ .Release.Name }}-vllm"
   {{- end }}
   {{- if .Values.LLM_MODEL_ID }}
+  # NOTE:
+  # delete LLM_MODEL once https://github.com/opea-project/GenAIComps/pull/1089 is merged
   LLM_MODEL: {{ .Values.LLM_MODEL_ID | quote}}
+  LLM_MODEL_ID: {{ .Values.LLM_MODEL_ID | quote}}
+  {{- end }}
+  {{- if contains "opea/llm-docsum" .Values.image.repository }}
+  MAX_INPUT_TOKENS: {{ .Values.MAX_INPUT_TOKENS | quote }}
+  MAX_TOTAL_TOKENS: {{ .Values.MAX_TOTAL_TOKENS | quote }}
```

Comment:
Why these limits should not be used for other images?

AFAIK DocSum normally needs / would benefit from more tokens than the other applications, so constraining token counts only for that seems wrong...

Reply:
I've already double checked other images' source code. Only llm-docsum-tgi and llm-docsum-vllm support these parameters.